nuget_witsml.md

```diff
@@ -1,9 +1,8 @@
 ## Access pre-built Witsml nuget package
 
 If you only need to include the [Witsml](https://github.com/equinor/witsml-explorer/tree/main/Src/Witsml) project
-```csharp
+```sh
 witsml-explorer/Src/Witsml
-```
 
```

Comment:
You still need the closing of the section. 
My suggestion did not include the "closing",  I am sorry for that.

Reply:
no problem, my eyes missed the dots :) I will fix, thanks!